### ISSUE TITLE
2019-11-20 tc010 M1SD test case variant with All FLGEXAUC set to 1, i.e. exclude all data points from AUC computations crashes computation engine

### ISSUE THREAD

**Issue by tensfeldt** [tensfeldt/openNCA#35]
This WAS working appropriately in current master branch, i.e. expectation is that all parameters other than AUCs are computed as expected. Requested AUCs are set to NA.

```
flags$FLGEXAUC <- rep(1, nrow(flags)) # exclude all records/data points from KEL calcs
> results_list <- run_computation(data=d, map=mct, flag=flags, parameterset=parameterset)
Error in value[[3L]](cond) : Error in tmp[[2]]: subscript out of bounds
For SDEID 2925168
In addition: Warning messages:
1: In run_M1_SD_computation(data = merged_data, map = map_data, method = method,  :
  Flag 'FLGACCEPTTAU' cannot be computed if 'TAUi' is not provided
2: In run_M1_SD_computation(data = merged_data, map = map_data, method = method,  :
 
 Show Traceback
 
 Rerun with Debug
 Error in value[[3L]](cond) : Error in tmp[[2]]: subscript out of bounds
For SDEID 2925168 
```

**Comment by jhhughes256** [comment]
### Update

I can replicate this error for all test cases involving a partial AUC (tc010, tc011 & tc015). It does not occur for tc012, tc013, tc014 or tc016. When partial AUCs are added to the mct of tc012, tc013,l tc014 (using historic commit 066fa5d8a1) or tc016 this error can be replicated.

When setting FLGEXAUC to 1 in tc013 without partial AUCs added a unique error is experienced outlined in issue #41 .

**Comment by jhhughes256** [comment]
### Update

It appears that this issue is strongly related to #6. tc013 can be run without error with the following changes from the base testcase:

```
mct$AUC.1.T1 <- 9
mct$AUC.1.T2 <- 505
mct$AUC.2.T1 <- 9
mct$AUC.2.T2 <- 720
mct$AUC.3.T1 <- 108
mct$AUC.3.T2 <- 505
mct$INCLUDEEXTRAPOLATION <- 0
flags$FLGEXAUC <- rep(1, nrow(flags))
```

The partial AUCs listed are interpolated points to extrapolated points. By turning off extrapolation the error no longer occurs. Therefore, it appears that with exclusion of all data points from being used to determine AUC, all partial AUC points are considered extrapolated points.

This makes this the same issue, partial AUCs for extrapolated to extrapolated points causing a "subscript out of bounds" error.

**Comment by rudraya** [comment]
This issue was addressed in 02b2470, please let me know if it still occurs.

**Comment by jhhughes256** [comment]
### Update

This no longer occurs for 02b2470.
